tapir/shifts/tests/test_shift_template_duplicate.py

```diff
@@ -107,6 +113,6 @@ def test_shiftTemplateDuplicateForm_originalTemplateNotDuplcated(self):
             follow=True,
         )
         self.assertEqual(response.status_code, 200)
-        self.assertNotEqual(ShiftTemplate.objects.count(), 2)
+        self.assertNotEqual(1, ShiftTemplate.objects.count())
```

Comment:
This should be `self.assertEqual` and not `self.assertNotEqual`, you would know if you had at least ran the test once.